apiary-receivers-parent/apiary-receiver-common/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.expedia.apiary</groupId>
+    <artifactId>apiary-receivers-parent</artifactId>
+    <version>1.3.3-SNAPSHOT</version>
```

Comment:
Let's go with a version bump to 1.4.0-SNAPSHOT since we're adding quite a bit of functionality with all these changes.

Reply:
Good plan. This will have to be changed in all modules right?

Reply:
Yep, thats done